contracts/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@iden3/contracts",
   "description": "Smart Contract library for Solidity",
-  "version": "2.4.2",
+  "version": "2.4.4",
```

Comment:
Why `2.4.4` instead of `2.4.3`?

Reply:
It's result of collecting of 3PRs into one, but true, let it be 2.4.3